maintainers/maintainer-list.nix

```diff
@@ -10888,4 +10888,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  wunderbrick = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add wunderbrick``.

Reply:
Done, thanks!

Reply:
I really should add to the template that this needs to be sorted alphabetically.

Reply:
> I really should add to the template that this needs to be sorted alphabetically.

Would you like me to change the maintainers so mine is in alphabetical order? The last few entries are in no particular order. Please tell me what specifically, if anything, I need to do to get this package merged.

Reply:
> Would you like me to change the maintainers so mine is in alphabetical order?

yes please.

> The last few entries are in no particular order.

thats by accident. They should be sorted, too.

Reply:
So I put my maintainer info in the set where it makes sense but I'm not sure how to actually sort the whole maintainer-list.nix so it's actually correct. Maybe someone with a better handle on Nix can do this in a different PR.